pythonFiles/tests/debug_adapter/test_ptvsd_folder_name.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import pytest
+import sys
+
+from os import path
+
+try:
+    from unittest.mock import patch, mock_open
+
+    from packaging.tags import sys_tags
+    from ptvsd_folder_name import ptvsd_folder_name
+except:  # Python 2.7
+    print("Not importing anything for Python 2.7 since the test will be skipped.")
+
+
+ROOT = path.dirname(path.dirname(path.dirname(path.dirname(path.abspath(__file__)))))
+PYTHONFILES = path.join(ROOT, "pythonFiles", "lib", "python")
+
+
+def isPython37():
+    return sys.version_info.major == 3 and sys.version_info.minor == 7
+
+
+def open_requirements_with_ptvsd():
+    return patch(
+        "ptvsd_folder_name.open", mock_open(read_data="jedi==0.15.1\nptvsd==5.0.0")
+    )
+
+
+def open_requirements_without_ptvsd():
+    return patch("ptvsd_folder_name.open", mock_open(read_data="jedi==0.15.1\n"))
+
+
+def mock_path_exists_true(pathname):
+    return True
+
+
+def mock_path_exists_false(pathname):
+    return False
+
+
+@pytest.mark.skipif(not isPython37(), reason="PTVSD wheels shipped for Python 3.7 only")
+class TestPtvsdFolderName(object):
```

Comment:
For Python 3 you don't have to explicitly put `(object)`.

```suggestion
class TestPtvsdFolderName:
```

I'm guessing you did this because we run our tests under Python 2.  In that case you should be able to skip the entire test module.  At the top of the file put the following:

```python
pytestmark = pytest.mark.skipif(
    sys.version_info[:2] != (3, 7),
    reason="PTVSD wheels shipped for Python 3.7 only")
```

or I would expect `unittest.SkipTest` to work as well:

```python
if sys.version_info[:2] != (3, 7):
    import unittest
    raise unittest.SkipTest("PTVSD wheels shipped for Python 3.7 only")
```

Either way, having one module-level skip makes `isPython37()` (or `IS_PY37`) unnecessary.